doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,120 @@
+
+These issues can be solved by creating the test data on-the-fly before running the test. Then the test-data is documented and under version control. That can be very helpful, to make sure that there were no changes on the test fixture between two runs.
+
+One downside of this approach is that it only works for rather small test files. For larger files, creating and uploading the files gets too expensive and becomes too slow. To solve this we added a cache. Before uploading the test files, we create a checksum and check if the files are already in place. Only if there were changes we re-upload the files.
+
+## Displaying Performance Tests Results
+
+In order to evaluate the test results, and discuss them, we need to visualize them. For that we decided on a [Redash](https://redash.io/) dashboard. Redash is an open source tool that allows you to create online dashboards for SQL queries. So we push our performance test results into an Exasol database and use Redash to evaluate them.
+
+## Setting Goals
+
+The first test run generated the following results:
+
+| Test Case                          | Time in Seconds |
+|------------------------------------|-----------|
+| testLoadManyParquetRows            | 23        |
+| testLoadLargeParquetRows           | 13.02     |
+| testLoadManyParquetColumns         | 12.9      |
+| testLoadManyParquetRowsFromOneFile | 74.64     |
+| testManySmallJsonFiles             | 685.87    |
+| testLoadFewBigSalesParquetFiles    | 662.28    |
+| testLoadSalesParquetFiles          | 219.21    |
+
+The first 4 cases (`testLoadManyParquetRows`, `testLoadLargeParquetRows`, `testLoadManyParquetColumns` `testLoadManyParquetRowsFromOneFile`) all load the same amount of data (1 GB). They just differ on how the data is distributed over rows, files and columns.
+
+This dataset consists of pseudo-random generated strings. The different test cases are built as show in the following table:
+
+| Test Case                          | String size in byte | Rows      | Files | Columns |
+|------------------------------------|---------------------|-----------|-------|---------|
+| testLoadManyParquetRowsFromOneFile | 1,000               | 1,000,000 | 1     | 1       |
+| testLoadManyParquetRows            | 100                 | 1000000   | 10    | 1       |
+| testLoadLargeParquetRows           | 1,000,000           | 100       | 10    | 1       |
+| testLoadManyParquetColumns         | 1000                | 1000      | 10    | 100     |
+
+You can observe that the product of all columns is always 1,000,000,000 so 1 GB.
+
+Similar the two trailing test split the same test data in 200 (`testLoadSalesParquetFiles`) or 8 `testLoadFewBigSalesParquetFiles`. There the data is exemplary sales data consisting of 7 decimal columns with 3,621,459,710 rows.
```

Comment:
```suggestion
The last two trailing tests use an exemplary sales data consisting of 7 decimal columns with 3,621,459,710 rows. These two tests split the the sales data into 8 (`testLoadFewBigSalesParquetFiles`) and 200 (`testLoadSalesParquetFiles`) files respectively.
```

Reply:
Reworked a little bit, because `the same test data` seemed like data for the first 4 tests.